Karen_dinner_party.rb

```diff
@@ -0,0 +1,65 @@
+
+#use for a pre-populated party with guest, food, and bevs. Otherwise use user input option below.
+#party_elements = {beverages: ["coke", "diet coke", "sprite", "tequila", "tequiza", "zima", "bud light lime", "orange juice", "water", "coffee", "tea", "jim beam", "cerveza"], food: ["porkchop", "veggie nuggets", "shrimp", "spaghetti"], people:
+#["Karen", "Wolfie", "Erin", "Tim", "Thom", "Stephen", "Ken", "James"], seat_num:[]}
+
+#user input arrays
+party_elements = {beverages: [], food: [], people:
+[], seat_num:[]}
+
+
+#puts "#{party_elements[:people].sample} will have the #{party_elements[:food].sample} to eat with #{party_elements[:food].sample} on the side. And to drink, #{party_elements[:beverages].sample}! Yum!"
+
+puts "Who would you like to invite to the party?"
+  array = party_elements[:people]
+  invite = gets.chomp
+  party_elements[:people].push(invite)
+  while
+    invite != ''
+    invite = gets.chomp
+    array.push invite
+  end
+  array.pop
```

Comment:
Instead of using `.pop` to remove the last (bad) input, you may want to just conditionally use `.push`:

```rb
if invite != ""
  party_elements[:people].push(invite)
end
```

Just less likely to cause bugs.